modules/nextflow/src/main/groovy/nextflow/conda/CondaCache.groovy

```diff
@@ -359,8 +359,8 @@ class CondaCache {
      *      The {@link DataflowVariable} which hold (and pull) the local image file
      */
     @PackageScope
-    DataflowVariable<Path> getLazyImagePath(String condaEnv) {
-
+    DataflowVariable<Path> getLazyImagePath(String rawCondaEnv) {
+        def condaEnv = normalizeCondaEnv(rawCondaEnv)
```

Comment:
Instead of introducing a new function, what about using [condaPrefixPath](https://github.com/nextflow-io/nextflow/blob/5462d7a6b3343c4513e2c16faac5de34d4756eda/modules/nextflow/src/main/groovy/nextflow/conda/CondaCache.groovy#L189-L189) and making sure it return an absolute  and normalized path ?

Reply:
I have implemented this change in PR [#5540](https://github.com/nextflow-io/nextflow/pull/5540)